tools/AzureRM/AzureRM.psd1

```diff
@@ -219,7 +191,7 @@ AzureRM.Websites
         # ExternalModuleDependencies = @()
 
     } # End of PSData hashtable
-
+    
  } # End of PrivateData hashtable
 
 # HelpInfo URI of this module
```

Comment:
@MiYanni I can't make a comment on the file, but it looks like changes were made to `AzureRM.psm1` that should be reverted. Would you mind checking that removing the beginning `/` character from the entry in the [`.gitignore`](https://github.com/Azure/azure-powershell/blob/preview/.gitignore#L6) file resolves the issue with changes being allowed to be checked in for this file?